src/Uno.UI/UI/Xaml/Controls/TreeView/TreeViewItemAutomationPeer.cs

```diff
@@ -0,0 +1,298 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Java.Lang.Annotation;
+using Uno.UI.Helpers.WinUI;
+using Windows.Graphics.Display;
+using Windows.UI.Xaml.Automation.Peers;
+using Windows.UI.Xaml.Automation.Provider;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+
+namespace Windows.UI.Xaml.Automation.Peers
+{
+	public partial class TreeViewItemAutomationPeer
+	{
+		private readonly TreeViewItem _owner;
+
+		public TreeViewItemAutomationPeer(TreeViewItem owner) : base(owner)
+		{
+		}
+
+		// IExpandCollapseProvider 
+		public ExpandCollapseState ExpandCollapseState
+		{
+			get
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a way to break out of this property accessor's recursion.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5156819)